build_tools/packaging/python/templates/rocm-sdk-core/src/rocm_sdk_core/_cli.py

```diff
@@ -9,10 +9,137 @@
 from ._dist_info import ALL_PACKAGES
 
 CORE_PACKAGE = ALL_PACKAGES["core"]
+CORE_PY_PACKAGE_NAME = CORE_PACKAGE.get_py_package_name()
+
+
+def _get_core_module_path():
+    # NOTE: dependent on there being an __init__.py in the core package.
+    core_module = importlib.import_module(CORE_PY_PACKAGE_NAME)
+    return Path(core_module.__file__).parent
+
+
+DEVEL_PACKAGE = ALL_PACKAGES["devel"]
+DEVEL_PURE_PY_PACKAGE_NAME = DEVEL_PACKAGE.pure_py_package_name
+DEVEL_PY_PACKAGE_NAME = DEVEL_PACKAGE.get_py_package_name()
```

Comment:
These changes are unrelated to Python 3.10 compatibility.

They came from https://github.com/ROCm/TheRock/pull/2428, which was reverted in https://github.com/ROCm/TheRock/pull/2472. Please fix your git workflow to not pull in unrelated changes to your own PRs.